utilities_for_test.go

```diff
@@ -323,3 +323,5 @@ func createFungibleToken(env *IntegrationTestEnv, opts ...TokenCreateTransaction
 	}
 	return *receipt.TokenID, err
 }
+
+const transferAmount = 100
```

Comment:
I think that values specific to a test should be defined in the tests themselves. I like the idea we have on reusing certain repeated transactions but I think this could potentially grow into having lots of specific constants in the utils for testing.

Reply:
Moved